dap_prinz_green_jobs/pipeline/ojo_application/ojo_sample/deduplication_utils.py

```diff
@@ -0,0 +1,139 @@
+"""
+Functions to deal with duplication in job adverts.
+
+get_deduplicated_job_adverts takes the job adverts (only job id and date columns are needed)
+and chunks up the dates and then removes duplicates from each chunk, before recombining.
+
+"""
+
+from datetime import datetime, timedelta, date
+from dateutil.relativedelta import relativedelta
+import random
+from hashlib import md5
+
+from dap_prinz_green_jobs import logger
+
+
+def short_hash(text):
+    """Generate a unique short hash for this string - from ojd_daps"""
```

Comment:
being pedantic but you could add types i.e.  
```suggestion
def short_hash(text:str) -> int:
    """Generate a unique short hash for this string - from ojd_daps"""
```

Reply:
I am so bad for this! but really want to start with good practices!